meta-resin-common/recipes-kernel/linux/kernel-devsrc.bbappend

```diff
@@ -0,0 +1,21 @@
+do_install_append() {
+   tar -czf ${B}/kernel_source.tar.gz -C "$kerneldir/../" .
+}
+
+inherit deploy
+addtask do_deploy before do_package after do_install
+
+do_deploy() {
+    cp ${B}/kernel_source.tar.gz ${DEPLOYDIR}/
+    rm ${B}/kernel_source.tar.gz
+}
+
+# Quite a few devices have random precompiled elf binaries in their
+# kernel src git trees. Remove various checks from this package to prevent
+# QA errors
+INSANE_SKIP_${PN} = "arch debug-files"
```

Comment:
This makes me feel uneasy but I don't really chase it because this archive doesn't end up on target. But still, why?!

Reply:
So the kernel-devsrc recipe creates a package. Various kernel trees in various devices have elfs/debug files in the kernel source tree itself.

It'll be too much hassle to go around in each device tree and add bbappends to remove those files. So I skipped these checks for the package itself.

Reply:
I understood that. It looks ugly to deploy debug symbols files in the src package. But anyway - for now as long as it an external artefact I'm fine. We should try to fix it in the future.